docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -244,5 +279,274 @@ function useMintlist({
+            <Form>
+              <div className="grid">
+                <FieldArray name="nfts">
+                  {({ insert }) => (
+                    <>
+                      {values.nfts.map((_, index) => (
+                        <div key={index}>
+                          <div className="mb-4">
+                            <LuxInputField
+                              placeholder="Name"
+                              name={`nfts.${index}.name`}
+                              required
+                            />
+                          </div>
+
+                          <ImageDropZone
+                            label="NFT Image"
+                            fieldName={`nfts.${index}.image`}
+                          />
+                        </div>
+                      ))}
+                      {values.nfts.length <= MAX_NFTS_PER_BATCH &&
+                        values.nfts.length < availableToUpload && (
+                          <button
+                            type="button"
+                            className="add-nft-button animated"
+                            onClick={() =>
+                              insert(values.nfts.length, {
+                                name: "",
+                                image: "",
+                              })
+                            }
+                          >
+                            <PlusIcon
+                              style={{
+                                width: "2rem",
+                                height: "2rem",
+                              }}
+                            />
+                          </button>
+                        )}
+                    </>
+                  )}
+                </FieldArray>
+              </div>
+              <div className="mt-4 flex-center spread">
+                <LuxSubmitButton
+                  label={`Upload ${values.nfts.length} NFT${
+                    values.nfts.length !== 1 ? "s" : ""
+                  }`}
+                  rounded
+                  color="brand"
+                  disabled={!(isValid && values.nfts.every((nft) => nft.image))}
+                />
+                <LuxButton
+                  label="Disconnect Wallet"
+                  onClick={onSignOut}
+                  color="secondary"
+                  size="small"
+                  variant="link"
+                />
+              </div>
+            </Form>
+          )}
+        </Formik>
+      </InteractiveWell>
+      <style jsx>{`
+        .grid {
+          display: grid;
+          grid-template-columns: repeat(3, 1fr);
+          column-gap: 1rem;
+          row-gap: 2rem;
+        }
+
+        .add-nft-button {
+          border: 1px solid var(--primary-border-color);
+          border-radius: var(--border-radius);
+          background-color: var(--faint-gray);
+          // Make sure the button doesn't collapse when the grid is empty.
+          min-height: 11.8rem;
```

Comment:
This feels kinda hacky / weird. Can we avoid doing this?

Reply:
Now as we don't have the "-" button anymore, I don't think we need this, good catch